README.asciidoc

```diff
@@ -91,6 +91,19 @@ you may modify later to match your vagrant requirements:
 $ vagrant origin-init --stage inst --os (fedora|centos7|rhel7) <instance name>
 ----
 
+If you are building a +bootstrap+ image to package and share with others, use the following:
+
+[source, sh]
+----
+$ vagrant origin-init --stage bootstrap --os (fedora|centos7|rhel7) <instance name> --local-source
+----
+NOTES:
+
+1. See link:#initial-setup[Initial Setup] below for a discussion on your choices for +--stage+.
+
+2. If you use the --local-source flag, a copy of the github origin repository will be installed in the image rather than sync'ed folders.
```

Comment:
The same concept came from V2 where we use install_from_local_source.  Do you feel I shorted it too much?